doc/classes/Node2D.xml

```diff
@@ -155,6 +155,10 @@
 		<member name="z_index" type="int" setter="set_z_index" getter="get_z_index" default="0">
 			Z index. Controls the order in which the nodes render. A node with a higher Z index will display in front of others.
 		</member>
+		<member name="y_sort_enabled" type="bool" setter="set_y_sort_enabled" getter="is_y_sort_enabled" default="true">
+			If [code]true[/code], child nodes are Y-sorted, otherwise Y-sorting is disabled. Y-sorting sorts all child nodes by their Y position. The child node must inherit from [CanvasItem] for it to be sorted. Nodes that have a higher Y position will be drawn later, so they will appear on top of nodes that have a lower Y position.
+			Nesting of Y-sorting nodes is possible. Children Y-sorted nodes will be sorted in the same space as the parent Y-sort, allowing to better organize a scene or divide it in multiple ones, yet keep the unique sorting.
```

Comment:
```suggestion
			If [code]true[/code], child nodes with the lowest Y position are drawn before those with a higher Y position. If [code]false[/code], Y-sorting is disabled. Y-sorting only affects children that inherit from [CanvasItem].
			You can nest nodes with Y-sorting. Children Y-sorted nodes are sorted in the same space as the parent Y-sort. This feature allows you to organize a scene better or divide it into multiple ones without changing your scene tree.
```

Reply:
This is a much better description, thank you! I was unsure what the best way to document the feature was, it is mostly just a copy of the original YSort docs, with slight modifications. I definitely agree that a manual describing the feature would be more clear than a member description. I will update the PR as soon as I can.

Reply:
I have updated the PR.